Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -16,6 +16,7 @@ final class StringExtensionsTests: XCTestCase {
     func testBase64Decoded() {
         XCTAssertEqual("SGVsbG8gV29ybGQh".base64Decoded, helloWorld)
         XCTAssertEqual("http://example.com/xxx", "aHR0cDovL2V4YW1wbGUuY29tL3h4eA".base64Decoded)
+        XCTAssertEqual(helloWorld, "SGVsbG\n8gV29ybGQh".base64Decoded)
```

Comment:
I meant to have both :)
```suggestion
        XCTAssertEqual(helloWorld, "SGVsbG\n8gV29ybGQh".base64Decoded)
        XCTAssertEqual(helloWorld, "SGVsbG8gV29ybGQh\n".base64Decoded)
```